projects/wavpack/project.yaml

```diff
@@ -5,5 +5,3 @@ auto_ccs:
 - thuanpv.nus@gmail.com
 sanitizers: 
 - address 
-- memory 
```

Comment:
please try to enable memory and undefined sanitizers later, they find nice set of bugs.

Reply:
Thanks. We have a compilation issue so we temporarily disable these sanitizers. Will definitely enable them again once the issue is fixed. With the address sanitizer, the fuzzer has already found some interesting bugs in our local setup.